hw0/Eric860730/countline.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
You do not need to turn in `hw0`.  And please remove unneeded file like this from your `hw1` submission.